in_toto/runlib.go

```diff
@@ -21,20 +19,31 @@ using sha256 and returns a map in the following format:
 If reading the file fails, the first return value is nil and the second return
 value is the error.
 */
+
 func RecordArtifact(path string) (map[string]interface{}, error) {
+
+	mapper := createMap()
```

Comment:
I believe that both `mapper` and `retMap` are not descriptive enough variable names in the context of  `recordArtifact`